src/shared/components/palette/case-flag/components/search-language-interpreter/search-language-interpreter.component.html

```diff
@@ -0,0 +1,35 @@
+<h2 class="govuk-heading-l">Search for a language interpreter</h2>
```

Comment:
@johnbenjamin-hmcts - minor observation: do you think it is worth moving the heading text into an enum, like we did for Global Search, instead of it being hard-coded in the HTML?

Reply:
@Daniel-Lam - I have created an enum as part of select flag type ticket as per your suggestion :)

Reply:
Fixed as part of the base branch https://github.com/hmcts/ccd-case-ui-toolkit/pull/1018